synfig-studio/images/CMakeLists.txt

```diff
@@ -255,13 +260,13 @@ foreach (IMAGE IN ITEMS ${IMAGES})
     endif()
     add_custom_command(
         TARGET build_images POST_BUILD
-        COMMAND synfig ${SRC}.sif --time ${TIME}f -w ${W} -h ${H} --quiet
-        COMMAND mv ${SRC}.png ${CMAKE_CURRENT_BINARY_DIR}/${IMAGE}.png
+        COMMAND synfig_bin ${SRC}.sif --time ${TIME}f -w ${W} -h ${H} --quiet
```

Comment:
This is exactly what I had mean. Awesome!

Reply:
:)